verifier.sh

```diff
@@ -375,6 +361,7 @@ _devtest_innervm_run () {
 
     repo_id="$GEM_GIT_REPO"
     ssh -t  $lxc_ip "git clone --depth=1 -b $branch_id $repo_id/$GEM_GIT_PACKAGE"
+    ssh -t  $lxc_ip "git clone --depth=1 -b $branch_id $repo_id/oq-moon || git clone --depth=1 -b  $repo_id/oq-moon"
```

Comment:
Why are you using `-b` in the `||` exception? Since you are not passing any branch but using master how it could work?

```
git clone --depth=1 -b https://github.com/gem/oq-moon
You must specify a repository to clone.

usage: git clone [<options>] [--] <repo> [<dir>]
```